### DISCUSSION TITLE
Update multiple items on an index?

### DISCUSSION THREAD

**Discussion by francoism90** [livewire/livewire#6994] [1. Help] [answered]
Imaging having an index page:
- Item (id 100)
- Item (id 200)
- Item (id 50)
- ..

I have an Filament panel, that updates Item id `200` with a new name, how can I push this change to the index?
I know it's possible to add listeners (Laravel Echo + Soketi) on each item, but this seems unwanted, especially since multiple items are being displayed, causing a lot of connections to be created.

On VueJS, I used a different trick. When someone opens the item, I simple replaced this item in the global 'index' store (find key => update). However on Livewire I don't think a global state manager exists (like Pinia)?

I could listen on a namespace: https://laravel.com/docs/10.x/broadcasting#namespaces, however you could potentially end up with records that aren't even on the index. Not a problem on small applications, but with hundred of records being updated, it may be unwanted as well.

Maybe I'm overthinking this, please let me know if it's possible with Livewire. :)

Thanks!

**Comment by joshhanley** [comment, marked as answer]
@francoism90 just dispatch an event from the panel component like "item-update" and add a listener for that on the index component. Should just work and not require Laravel Echo.

Hope this helps!

**Reply by francoism90** [reply]
@joshhanley Thanks for your reply! Does this work when doing something like this:
- /home
- /posts/item/100 (opened on /home)

How would one update the /home item?
I'm now using wire.poll om /home, this seems to work as well, it also refreshes when updating this on let's say an API.
But as I understood this is much more traffic usage, compared to a websocket?

Edit: with your method, wouldn't I end up with multiple listeners as well? Or is this not a problem?
